Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name    = "inotify"
-version = "0.8.3"
+version = "0.9.0"
```

Comment:
Can you please remove this change? I prefer to update the version right before the release, for the following reasons:
- Changing this in pull requests is not a reliable process, as not all contributors do that. We could make it a requirement, but that would put more of a burden on contributors, and also on me, because I'd have to remember to keep track of that.
- Changing the version makes the process of using `[patch]` in Cargo.toml to replace the latest `inotify` release with a Git version more complicated, as `[patch]` only works if the versions are the same.

Reply:
Sure thing.

Reply:
Thank you!